Casamari/MNC010.xml

```diff
@@ -122,77 +215,163 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            </list>
                         </collation>
                         <condition key="good">
-                           The manuscript was resewn. The original boards and the leather cover have been possibly restored in Italy, as shown by some elements 
-                           present in the binding (presence of papers, non-Ethiopian endbands). Two pairs of sewing stations have been added during the restoration process.
-                        </condition>
+                           <!--TO BE CHECKED BY MARCO:-->
+                           <p>The manuscript was resewn. The original boards and the leather cover
+                              have been possibly restored in Italy, as shown by some elements
+                              present in the binding (presence of papers, non-Ethiopian endbands).
+                              The book concervators used the original material for restoraion and
+                              added problay new material which was lost (endbands). Quire 1 (<ref
+                                 target="#q2"/>), 3 (<ref target="#q4"/>), and 4 (<ref target="#q5"
```

Comment:
I think these references to quires might be reformulated depending on the printed visualization, in order to avoid reduplication. In other words: I frankly do not know how `<ref target="#q4"/>` will be printed, but the whole sequence might become something like `3 (quire 3), and 4 (quire 4)`.
I would say `<ref target="#q2 #q3 #q4"/> were possibly reinforced with parchment guards.` or something similar.